Support/Multitenancy/Multiple-Ironic-conductors/install-ironic.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -eux
+
+# shellcheck disable=SC1091
+. ./config.sh
+
+__dir__=$(realpath "$(dirname "$0")")
+
+IRONIC_USERNAME="$(tr -dc 'a-zA-Z0-9' </dev/urandom | fold -w 12 | head -n 1)"
```

Comment:
```suggestion
IRONIC_USERNAME="$(uuid-gen)"
```
We've been using uuid-gen for long time now as it doesn't have issues with entropy etc.